DEV: Fix warning about already initialized constants

gschlager opened the pull request.

tgxworld reviewed (changes requested):
I know alot of places in our code base does this but I've always felt that this fix isn't right. Instead, we should try and figure out why the class is being loaded twice. 

This commit I made in the past might help: https://github.com/discourse/discourse/commit/069a109cbb8745ceb29035b2317c1b115052a7e0

gschlager commented:
@tgxworld I updated the PR. This fixes it for me as I've only seen the warnings in my dev environment. The piece of code I removed doesn't seem to be needed anymore.

tgxworld commented:
@gschlager Thank you! The fix looks good to me :+1:
